website/src/main.tsx

```diff
@@ -0,0 +1,80 @@
+import ReactDOM from 'react-dom/client'
+import {
+  createHashRouter,
+  RouterProvider,
+  Outlet,
+} from "react-router-dom";
+import Playground from "./Playground";
+import Homepage from './Homepage.tsx'
+import "normalize.css";
+import "./main.css"
+import styled from "styled-components";
+import { useState } from 'react';
+import { Button } from './Buttons.tsx';
+import PackageDocsPage from './PackageDocsPage.tsx';
+import Guide from './Guide.tsx';
+
+const DebugMessage = styled.div<{ bg?: string, color?: string }>`
+padding: 0.5rem;
+padding-left: 1rem;
+display: flex;
+align-items: center;
+justify-content: space-between;
+background: ${(props) => props.bg ?? "#e2e2e2"};
+z-index:1000;
+color: ${(props) => props.color ?? "inherit"};
+
+& a {
+  color: inherit;
+}
+`;
+
+function PrPrompt() {
+  const [hidden, setHidden] = useState(false);
+
+  const location = window.location;
+  const regex = /pr-preview\/pr-(\d+)/;
+  console.log(location.href);
+  const match = location.href.match(regex);
+  if (match !== null) {
+    return hidden ? <></> : <DebugMessage>
+      <a href={`https://github.com/modmark-org/modmark/pull/${match[1]}`}>Preview of PR #{match[1]}</a>
```

Comment:
I love this message! But for some reason, I think the styling is wrong; when you go to the playground without this header you can't scroll (like, you can scroll in the text fields but not scroll down the entire page, which is reasonable), but this header makes it possible to "scroll away" the header. I don't know the reason for this, but maybe the page is as high as the monitor, regardless of if there is a bar or not?